PowerEditor/src/ScintillaComponent/columnEditor.cpp

```diff
@@ -472,6 +473,18 @@ void ColumnEditorDlg::switchTo(bool toText)
 
 	::SetFocus(toText?hText:hNum);
 
+	if (toText) 
+	{
+		const int stringSize = 1024;
+		TCHAR str[stringSize];

```

Comment:
`const int stringSize = 1024;` can be replaced with `constexpr int stringSize = 1024;`
Add initializer to be sure e.g. `TCHAR str[stringSize]{};`

Reply:
@ozone10 
good remark.
Could you add the "best practice" in contribution coding style:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#coding-style 
Or create a issue for the "best practice" so I can add them in the guideline?